plans/feat-1821-update-recheck.md

```
@@ -0,0 +1,103 @@
+
+## The problem, as measured
+
+`refreshUpdateStatus` is called exactly once, from the `app.listen` callback in
+`server/index.ts`. Nothing re-runs it. The UI's polling loop (`useUpdateStatus`) is not a
+refresh either — it stops the moment `ready` turns true, because until now the answer could
+not change without a restart.
+
+That makes the check structurally blind to the install method the README recommends. Verified
+on `a71872a1` by running the real functions:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)